terraform/README.md

```diff
@@ -0,0 +1,17 @@
+# Terraform
+
+This folder contains all code and resources required for the setup of AWS resources using terraform
+
+## Set-up and installation instructions
+
+1. Requires environment variables in a terraform.tfvars file:
+
+- access_key
```

Comment:
could we perhaps change the formatting of this to match the other readme files?

Reply:
I agree let's aim to maintain consistency where we can.